result/request.yaml

```diff
@@ -0,0 +1,77 @@
+openapi: 3.1.0
+
+info:
+  title: Request response models
+  description: >-
+    All standard request response schemas
+  version: ^0.0.0
+
+components:
+  responses:
+    SuccessNoContent:
```

Comment:
We did have lengthy discussions after which we came up with sync approach (which can be later extended to async approach with little modification).

But, it's still not clear why we stuck to 204 instead of 200 ? One very important scenario (among numerous others) where returning response body makes sense is when we need to return warning. e.g.
- User configures a flow with Ether/VLAN/IP/TCP and gave a frame size 64B (because that's a valid scenario)
- But internally, we might need to add instrumentation, which will `require adjusting the size to >64B`
- In response, user doesn't care about the internals, but `must be` informed about the adjustments in form of a warning (otherwise they'll just be confused looking at the increased frame size in results)